### PR TITLE
Update GH action versions

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request
Missed a mirror or two

https://github.com/yogstation13/DreamAnnotate/discussions/1
https://github.com/tgstation/tgstation/pull/62400
https://github.com/tgstation/tgstation/pull/70463
https://github.com/tgstation/tgstation/pull/70297
https://github.com/tgstation/tgstation/pull/70490

### CODE REVIEW

**Comment by Tastyfish** [comment]
The PR to update all of the actions was just merged upstream, either the bot hasn't gotten it yet or it's missed https://github.com/tgstation/tgstation/pull/70463

**Comment by Zonespace27** [comment]
> The PR to update all of the actions was just merged upstream, either the bot hasn't gotten it yet or it's missed [tgstation/tgstation#70463](https://github.com/tgstation/tgstation/pull/70463)

Never mirrored so give me a minute to manually do so

**Comment by Tastyfish** [comment]
For that matter, it also missed https://github.com/tgstation/tgstation/pull/70490 so if you just want to reset the entire file that should work.

**Comment by Zonespace27** [comment]
> For that matter, it also missed [tgstation/tgstation#70490](https://github.com/tgstation/tgstation/pull/70490) so if you just want to reset the entire file that should work.

Jesus I didn't realize we were that far behind, I reset everything

File: .github/workflows/compile_changelogs.yml [hunk 1]
```diff
@@ -49,4 +51,4 @@ jobs:
         if: steps.value_holder.outputs.ACTIONS_ENABLED
         uses: ad-m/github-push-action@master
         with:
-          github_token: ${{ secrets.SKYRATBOT_TOKEN }}
```
Comment (Tastyfish) [inline .github/workflows/compile_changelogs.yml]: This is probably intentionally SKYRATBOT_TOKEN. Should probably verify

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -4,20 +4,27 @@ on:
...
-        days-before-issue-stale: -1
-        days-before-issue-close: -1
+        days-before-close: 7
         stale-pr-label: 'Stale'
-        exempt-pr-labels: 'Good First PR,Upstream PR Merged,Stale-b-gone,Test Merged'
+        days-before-issue-stale: -1
+        stale-issue-label: 'Cleanup Flagged'
+        remove-issue-stale-when-updated: false
+        exempt-pr-labels: 'RED LABEL,Good First PR'
         operations-per-run: 300
```
Comment (Tastyfish) [inline .github/workflows/stale.yml]: These differences are probably intentional to our repo.

**Review by Tastyfish — COMMENTED** [review]
A couple of the diffs from hard resetting the files seem to be intentional differences with Skyrat

File: .github/workflows/stale.yml (hunk 2 above)
Comment (Zonespace27) [reply .github/workflows/stale.yml]: Talking to golden about this one

Comment (Zonespace27) [reply .github/workflows/stale.yml]: Readded labelling, the rest can stay

**Review by Tastyfish — APPROVED** [review]
I think it's all good. I'm not sure if the test merge bot is set up secrets-wise but that's separate from adding the workflow.

File: .github/workflows/codeowner_reviews.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        id: CodeOwnersParser
+        uses: tgstation/CodeOwnersParser@v1
+
+      #Request reviews
+      - name: Request reviews
+        if: steps.CodeOwnersParser.outputs.owners != ''
+        uses: tgstation/RequestReviewFromUser@v1
+        with:
+          separator: ' '
+          users: ${{ steps.CodeOwnersParser.outputs.owners }}
```
Comment (GoldenAlpharex) [inline .github/workflows/codeowner_reviews.yml:26]: Lmao I noticed we didn't have that, I just figured it was a quirk of being a downstream, hah.

File: .github/workflows/stale.yml [hunk 4]
```diff
@@ -4,20 +4,27 @@ on:
...
+        stale-pr-message: "This PR has been inactive for long enough to be automatically marked as stale. This means it is at risk of being auto closed in ~ 7 days, please address any outstanding review items and ensure your PR is finished, if these are all true and you are auto-staled anyway, you need to actively ask maintainers if your PR will be merged. Once you have done any of the previous actions then you should request a maintainer remove the stale label on your PR, to reset the stale timer. If you feel no maintainer will respond in that time, you may wish to close this PR youself, while you seek maintainer comment, as you will then be able to reopen the PR yourself"
```
Comment (GoldenAlpharex) [inline .github/workflows/stale.yml]: That missing period bugged me a lot and that's why I had added it ;)
```suggestion
        stale-pr-message: "This PR has been inactive for long enough to be automatically marked as stale. This means it is at risk of being auto closed in ~ 7 days, please address any outstanding review items and ensure your PR is finished, if these are all true and you are auto-staled anyway, you need to actively ask maintainers if your PR will be merged. Once you have done any of the previous actions then you should request a maintainer remove the stale label on your PR, to reset the stale timer. If you feel no maintainer will respond in that time, you may wish to close this PR youself, while you seek maintainer comment, as you will then be able to reopen the PR yourself."
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Thanks for doing the cleanup, it's appreciated!

**Review by GoldenAlpharex — APPROVED** [review]
Should be good now!

**Comment by Tastyfish** [comment]
The 1 failure is that stupid ice demon bug.

**Review by tf-4 — APPROVED** [review]

**Pull request merged**